src/backend/serverless/README.md

```diff
@@ -99,44 +98,46 @@ node ./deploy.js \
   -D 7200
 ```
 #### Transcoder Serverless stack
- Once you run ```deploy.js``` we will also execute deployment of Transcoder Serverless Application that contains resources for building an application that broadcast media from any URL of meeting sessions to a RTMP endpiont URL. Included is a Docker image and a serverless AWS CloudFormation template that you can deploy to your AWS account. The AWS CloudFormation template orchestrates resources (including Amazon Lambda and Amazon ECS) that run the broadcast application. When deployed, you can create Lambda test events to start/stop broadcast task in ECS. [checkout for more information or for custom deployment of transcoder app](../../../transcoding/README.md)
+Once you run ```deploy.js``` we will also execute deployment of Transcoder Serverless Application that contains resources for building an application that broadcast media from any URL of meeting sessions to a RTMP endpiont URL. Included is a Docker image and a serverless AWS CloudFormation template that you can deploy to your AWS account. The AWS CloudFormation template orchestrates resources (including Amazon Lambda and Amazon ECS) that run the broadcast application.
+> NOTE: If the backend deployment is successful there is no need to separately deploy the transcoder application but you still need to run the ECS task for broadcasting.
 
-## Outputs
+## Deployment Outputs
 Upon stack creation, CloudFormation will output the CloudFront distribution name. This domain name should be aliased by the Domain Name passed to the deployment script in order for a Domain Name deployment to function correctly:
 
-- `CloudFrontDomainName`
+- `CloudFrontDomainName`: The CloudFront distribution domain name which would be aliased by the actual domain name
 
 It will also output the app URLs for the Attendee, Moderator, and Talent web apps under the following keys respectively:
 
-- `AttendeeURL`
-- `ModeratorURL`
-- `TalentURL`
+- `AttendeeURL`: The URL for accessing the Attendee web app
+- `ModeratorURL`: The URL for accessing the Moderator web app
+- `TalentURL`: The URL for accessing the Talent web app
 
 Additionally, the URL for the backend API Gateway REST API and the URIs for the 2 API Gateway WebSockets. These can be used as inputs to other templates which might extend on the existing one.
 
-- `ApiURL`
-- `MessagingWebSocketURI`
-- `HandRaiseWebSocketURI`
+- `ApiURL`: API endpoint URL for Prod environment
+- `MessagingWebSocketURI`: The Messaging WSS Protocol URI to connect to
+- `HandRaiseWebSocketURI`: The HandRaise WSS Protocol URI to connect to
 
 The stack outputs the channel ID of the Elemental MediaLive instance that will be used to consume broadcast stream input and encode it. This is so you can find it easily when you need to start it just prior to beginning your live event's start:
```

Comment:
Can we mention that this needs to be started by going to the MediaLive console in AWS and clicking on 'Start' - alternatively you can use the --start-channel param in the deploy script to do this automatically.

Reply:
It is already mentioned in the section above: [https://github.com/aws-samples/amazon-chime-live-events/blob/update-documentation/src/backend/serverless/README.md?plain=1#L52](https://github.com/aws-samples/amazon-chime-live-events/blob/update-documentation/src/backend/serverless/README.md?plain=1#L52)